examples/walkthrough/README.md

```diff
@@ -1,73 +1,149 @@
 # Kubernetes 101 - Walkthrough
 
+## Kubectl CLI
+
+The easiest way to interact with Kubernetes is via the command-line interface.
+
+If you downloaded a pre-compiled release, kubectl should be under `platforms/<os>/<arch>`.
+
+If you built from source, kubectl should be either under `_output/local/bin/<os>/<arch>` or `_output/dockerized/bin/<os>/<arch>`.
+
+### Install
+
+Do one of the following to install kubectl:
+- Add the kubectl binary to your PATH
+- Move kubectl into a dir already in PATH (like `/usr/local/bin`)
+- Use `./cluster/kubectl.sh` instead of kubectl. It will auto-detect the location of kubectl and proxy commands to it.
```

Comment:
Can we revise lines 12-16, so that whats at line 16 is moved above the install options (16 its not really "install" but more of an alternative):

####Installing kubectl
You can use the version of kubectl thats included in the directory where you installed Kubernetes or install kubectl by adding it to your PATH environment variable.

 -  Run the kubectl commands by using `./cluster/kubectl.sh`, for example:  
    `$ ./cluster/kubectl.sh cluster-info`  
    (*Note: It will auto-detect the location of kubectl and proxy commands to it.*)
 - Install kubectl by adding it to PATH so that you can simply use 'kubectl', for example:  
    `$ kubectl cluster-info`